spring-smtp-gateway/smtp-gateway/src/main/java/com/java/example/smtpmq/gateway/streams/SMTPMailMessageConverter.java

```diff
@@ -0,0 +1,184 @@
+/* 
+Copyright (c) 2010, NHIN Direct Project
```

Comment:
this looks odd if this is now part of the sample source, also what is the license for this code?

Reply:
This needs to be removed.  HNIND is not an actual entity but a place holder for something that never formed.  I'm the actual author and owner of all this code, so I need to remove these.  Thanks for catching this.